src/js/candidate-detail.js

```diff
@@ -0,0 +1,19 @@
+let urlParams = new URLSearchParams(window.location.search);
```

Comment:
The `URLSearchParams` API [appears to have pretty narrow browser support](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Browser_compatibility). I don't feel great about relying on it.

Reply:
Yes, I realized that. We can however, replace it with our own custom method - the implementations in this PR are more of a "proof of concept" if you will, from our discussion from yesterday and they are subjected to changes as we see fit.

Reply:
Gotcha. As far as it being a PoC goes, I'm good with it